CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Minor: 7TV badges now automatically update upon changing. (#4512)
 - Minor: Stream status requests are now batched. (#4713)
 - Minor: Added `/c2-theme-autoreload` command to automatically reload a custom theme. This is useful for when you're developing your own theme. (#4718)
+- Minor: Populate mentions tab with messages from history. (#4475)
```

Comment:
```suggestion
- Minor: Mentions loaded from message history will now appear in the /mentions tab. (#4475)
```
explains what the feature actually does better for users

Reply:
Maybe replace message history with historical messages